_data/projects/TJ-Bot.yml

```diff
@@ -0,0 +1,11 @@
+name: TJ-Bot
+desc: TJ-Bot is a Discord Bot used on the Together Java server.
+site: https://github.com/Together-Java/TJ-Bot
+tags:
+  - java
+  - bot
+  - discord
+  - beginner-friendly
+upforgrabs:
+  name: help wanted
+  link: https://github.com/Together-Java/TJ-Bot/labels/help%20wanted
```

Comment:
@marko-radosavljevic Please use this instead https://github.com/Together-Java/TJ-Bot/labels/help%20wanted

Reply:
Currently only have phone available, but it seems already correct to me. ^^

@shiftkey mentioned in other PRs that the bot likely fails due to a bug:
https://github.com/up-for-grabs/up-for-grabs.net/pull/3330#issuecomment-1228885093

Reply:
Got it, seems to be that only.